src/modules/twap/services/extensibleFallbackSetupTxs.ts

```diff
@@ -0,0 +1,34 @@
+import { ExtensibleFallbackContext } from '../hooks/useExtensibleFallbackContext'
+import { getSignatureVerifierContract } from './getSignatureVerifierContract'
+import { COMPOSABLE_COW_ADDRESS, SAFE_EXTENSIBLE_HANDLER_ADDRESS } from 'modules/advancedOrders'
+import { MetaTransactionData } from '@safe-global/safe-core-sdk-types'
+
+export async function extensibleFallbackSetupTxs(context: ExtensibleFallbackContext): Promise<MetaTransactionData[]> {
+  const { chainId, safeAppsSdk, settlementContract } = context
+
+  const { safeAddress } = await safeAppsSdk.safe.getInfo()
+  const domainSeparator = await settlementContract.callStatic.domainSeparator()
+
+  const signatureVerifierContract = await getSignatureVerifierContract(context)
+  const composableCowContractAddress = COMPOSABLE_COW_ADDRESS[chainId]
+  const extensibleHandlerAddress = SAFE_EXTENSIBLE_HANDLER_ADDRESS[chainId]
+
+  const setFallbackHandlerTx = {
+    to: safeAddress,
+    data: signatureVerifierContract.interface.encodeFunctionData('setFallbackHandler', [extensibleHandlerAddress]),
```

Comment:
Same suggestion as the previous PR

Encode it using `populateTransaction`: https://github.com/cowprotocol/cowswap/blob/ed50eb839629d7df29a404a5d4663b56330d1900/src/modules/operations/bundle/buildPresignTx.ts#L12-L11

Reply:
same answer :)